### DISCUSSION TITLE
No pulls commited to the main

### DISCUSSION THREAD

**Discussion by AmadeusGraves** [IceWhaleTech/CasaOS#400] [General]
Hi, i see a lot of translations updates to UI but they are not accepted.  They are anyone that can review and pull them?

https://github.com/IceWhaleTech/CasaOS-UI/pulls

**Comment by learn-more** [comment]
Translations can be sent to https://github.com/IceWhaleTech/CasaOS-i18n

**Reply by AmadeusGraves** [reply]
thanks already pull spanish translation in there.
